product_multi_link/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Product Multi Link
+==================
+
+This module adds links between products:
+
+- cross-selling
+- up-selling
+- related
+
+These types of links are common in e-commerce shops.
+
+It can be used as a base to implement synchronizations with
+e-commerce (for instance, it is used in magentoerpconnect).
+
+Usage
+=====
+
+To use this module, you need to:
```

Comment:
This is intended as a template for the usage instructions. If you don't put instructions (although it's better to put them), please remove this paragrah